docs/remote-config/_get-started.md

```diff
@@ -133,6 +133,24 @@ smooth experience for your user, such as the next time that the user opens your
 app. See [Remote Config loading strategies](/docs/remote-config/loading)
 for more information and examples.
 
+## Step 7: Listen for updates in real time
```

Comment:
```suggestion
## Step 7: Listen to real-time updates
```

Reply:
Both Android and iOS are using this title: https://firebase.google.com/docs/remote-config/get-started?platform=android#add-real-time-listener